Library/Formula/camlp5.rb

```diff
@@ -1,8 +1,11 @@
 class Camlp5 < Formula
   desc "Preprocessor and pretty-printer for OCaml"
   homepage "http://camlp5.gforge.inria.fr/"
-  url "http://camlp5.gforge.inria.fr/distrib/src/camlp5-6.14.tgz"
-  sha256 "09f9ed12893d2ec39c88106af2306865c966096bedce0250f2fe52b67d2480e2"
+  url "https://ftp.ucsb.edu/pub/mirrors/linux/gentoo/distfiles/camlp5-6.15.tgz"
+  mirror "https://mirror.csclub.uwaterloo.ca/gentoo-distfiles/distfiles/camlp5-6.15.tgz"
+  mirror "https://mirror.netcologne.de/gentoo/distfiles/camlp5-6.15.tgz"
```

Comment:
We use the canonical upstream URL, even if it lacks SSL/TLS, unless it is broken or problematic in some other way.